test/selenium/selenium-base.py

```diff
@@ -33,6 +33,9 @@ def test30ChangeTabServices(self):
         self.wait_frame("services")
         self.wait_id("services-list")
         self.click(self.wait_text("Socket", cond=clickable))
+        if os.environ.get("BROWSER") == 'edge':
+            # HACK: Edge does not see elements (kdump menu entry) which are below the visible window area
+            self.driver.execute_script('document.getElementById("systemd-udevd.service").scrollIntoView()')
```

Comment:
heh, from screenshots, the page is on `sockets` but this should be in `services`. So somehow this is on a wrong tab

Reply:
oh, it is on a right page, but the id should be `systemd-udevd-kernel.socket` or `systemd-udevd-control.socket`

Reply:
D'oh, thanks! Pushed